docs/reference/filebeat/filebeat-input-entity-analytics.md

```diff
@@ -0,0 +1,997 @@
+---
```

Comment:
The `8.x` branches are using the asciidoc documentation, the documentation part of this PR needs to be manually backported to the corresponding asciidoc file.

Reply:
Yes, I'm aware of that.